static/js/public/snap-details/blog-posts.js

```diff
@@ -36,7 +36,7 @@ class BlogPosts {
       then(response => response.json()).
```

Comment:
This way of writing chained calls is confusing. It looks like a separate function call. But as it's not related to this PR lets wait till we turn prettier checks. They should catch those I guess.